RecoLocalTracker/SiPixelRecHits/src/SiPixelTemplateSplit.cc

```diff
@@ -47,7 +47,7 @@
 #include "FWCore/MessageLogger/interface/MessageLogger.h"
 #define LOGERROR(x) edm::LogError(x)
 #define LOGDEBUG(x) LogDebug(x)
-static int theVerboseLevel = 2;
+static constexpr int theVerboseLevel = 2;
```

Comment:
that's the role of "anonymous namespace" in C++  (w/r/t static as "local-to-file" in C)